types/address.go

```diff
@@ -110,9 +111,7 @@ func VerifyAddressFormat(bz []byte) error {
 	if verifier != nil {
 		return verifier(bz)
 	}
-	if len(bz) != AddrLen {
-		return fmt.Errorf("incorrect address length (expected: %d, actual: %d)", AddrLen, len(bz))
-	}
+
```

Comment:
We should check for max address length here

Reply:
Also is 0 ever a valid address length? If not we can have max length 256.

Reply:
0 is not a valid address length. However I believe this is a case where code simplicity (we're length-prefixing, not (length-1)-prefixing) trumps over performance (1-byte gain, 255 should be plenty already).